.travis.yml

```diff
@@ -103,6 +103,9 @@ before_install:
       sed -i.bak '/^coala/d' requirements.txt
       sed -i.bak '/^mypy-lang/d' requirements.txt bear-requirements.txt
     fi
+  # On ubuntu:14.04 this seems the best way to install `phpcbf`
+  # With ubuntu:16.04 `php-codesniffer` contains `phpcbf`
+  - sudo pear install PHP_CodeSniffer
```

Comment:
can you use the newer composer here?

also, you have to put this in a new shell script, `.ci/deps.composer.sh`, look at other package manager shell scripts to see how it is done

Reply:
but still this needs to be moved preferably to `.ci/deps.sh`